content/en/docs/tasks/administer-cluster/access-cluster-api.md

```diff
@@ -156,8 +156,7 @@ with future high-availability support.
 
 ### Programmatic access to the API
 
-Kubernetes officially supports client libraries for [Go](#go-client) and
-[Python](#python-client).
+Kubernetes officially supports client libraries for [Go](#go-client), [Python](#python-client), [Java](#java-client), [Dotnet](#dotnet-client), [Javascript](#javascript-client) and [Haskell](#haskell-client).
```

Comment:
Line 159: Add a comma before `and`:
```[Dotnet](#dotnet-client), [Javascript](#javascript-client), and [Haskell](#haskell-client).```
Line 159: nit: Should Dotnet be capitalized? Check other places on the page for consistency.
Line 219: The shell command is not rendering correctly. Adding a new line at line 219 should fix that.

Duplicate code:
The code examples are found at, https://github.com/kubernetes-client/java, https://github.com/kubernetes-client/csharp, https://github.com/kubernetes-client/javascript, https://github.com/kubernetes-client/haskell.
Should this page include links to those code examples instead of repeating in line?

Reply:
@kbhawkey Thanks for review. I am new to k8s community so sorry for making silly mistakes. 
There is no reason to remove link to [https://kubernetes.io/docs/reference/using-api/client-libraries/](https://kubernetes.io/docs/reference/using-api/client-libraries/) as there are other community maintained libraries present which are not official. This page only have details of official libraries. **So link to client libraries page should be present**.

I didnt understand Duplicate code point. I mean this page does have links to code examples .
Can you pls clear this point?

Reply:
hello @shub-asa1. 
Instead of repeating the code examples for each client from https://github.com/kubernetes-client/, could this page link to the examples maintained in the kubernetes-client repo?
One example of duplicate code:
- https://github.com/kubernetes-client/haskell/blob/master/kubernetes-client/example/App.hs
- https://deploy-preview-16940--kubernetes-io-master-staging.netlify.com/docs/tasks/administer-cluster/access-cluster-api#haskell-client

Reply:
@kbhawkey  code examples that are added are simple ones and it will be easy for new users to start testing from a simple program like to list all pods in a cluster. In [https://github.com/kubernetes-client/](https://github.com/kubernetes-client/) there are too many examples and it will be hard for new users to try these libraries for the first time. I mean these are simplest code examples which is enough to get new users start using client libraries.